tests/unit_tests/test_simulator.py

```diff
@@ -41,37 +41,42 @@ def corsika_file():
 
 
 @pytest.fixture
-def array_simulator(label, array_config_data, io_handler, db_config, simtel_path):
+def array_simulator(label, array_config_data, io_handler, db_config, model_version, simtel_path):
     array_simulator = Simulator(
```

Comment:
Probably also better to stick to ordering here:

```
simulator,
simulator_source_path,
label=None, etc.
```

Reply:
Fixed.